src/main/java/org/broadinstitute/consent/http/models/DataUse.java

```diff
@@ -22,6 +22,7 @@ public class DataUse {
     private Boolean hmbResearch;
     private List<String> diseaseRestrictions;
     private Boolean populationOriginsAncestry;
+    private Boolean openAccess;
```

Comment:
This should not be part of DataUse - in essence, it is a dataset property, not a use restriction condition that we need for doing a logical match between a DAR and a Dataset.

Reply:
I made this a dataset property. This PR is now getting pretty big, I am trying to think of a good way to split it up but I am not sure there is.